package.json

```diff
@@ -2,47 +2,94 @@
   "name": "little-window",
   "version": "1.0.0",
   "description": "A chatbot featured on Chayn websites, which quickly directs users to Chayn’s relevant internal or external resources",
-  "main": "config.js",
+  "author": "Chayn",
+  "license": "ISC",
   "scripts": {
-    "start": "node ./src/server.js",
-    "start:test": "NODE_ENV=production node ./src/server.js",
-    "postinstall": "cd client && npm install && npm run build",
-    "test": "NODE_ENV=test tape ./src/test/*test.js | tap-spec && cd client && npm run test",
-    "server": "NODE_ENV=dev nodemon ./src/server.js",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
-    "client": "cd client && npm start",
-    "lint": "eslint '**/*.js'"
+    "prebuild": "rimraf dist",
+    "postinstall": "nest build",
```

Comment:
It's worth still having `"build": "nest build"` as a separate task, and then this postinstall can just do `npm build`. (Just so it's clear it's not Heroku specific).

Reply:
Updated!